### PR TITLE
Create ui-c8y-1018-503-73-improve-locators-for-users-cy-spec-5872.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by amio

Ticket: [MTM-58154](https://cumulocity.atlassian.net/browse/MTM-58154)
Original PR: [5872](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5872)

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1018-503-73-improve-locators-for-users-cy-spec-5872.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: improve locators for users.cy spec (#5872)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-73-improve-locators-for-users-cy-spec-5872.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved locators in users Cypress spec
```

File: content/change-logs/platform-services/ui-c8y-1018-503-73-improve-locators-for-users-cy-spec-5872.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58154
+version: 1018.503.73
+---
+improve locators for users.cy spec (#5872)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1018-503-73-improve-locators-for-users-cy-spec-5872.md:17]: The Claude AI suggested to use the following body:
```suggestion
The Cypress tests for the users management area were using non-optimal locators to identify elements on the page, making the tests brittle and hard to maintain. The locators have been improved to be more specific and resilient to changes in the application. This change makes the users Cypress spec more stable and reliable, reducing false positives in the test runs.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@Airdroplet I assume that this PR can be closed?

**Comment by Airdroplet** [comment]
> @Airdroplet I assume that this PR can be closed?

yes, it's not about change in product but rather internal. Closing.

**Pull request closed without merging**